frontend/src/pages/CreatePost.jsx

```
@@ -0,0 +1,439 @@
+          <h2 className="text-2xl font-bold tracking-wide flex items-center gap-2">
+            <Check className="w-7 h-7" /> Post Created Successfully
+          </h2>
+        </div>
+        <p className="text-gray-700 mb-6 text-center">
+          Your job post has been created and is now live. Would you like to view your posts?
+        </p>
+        <div className="flex justify-center space-x-4">
+          <button
+            onClick={onConfirm}
+            className="bg-gradient-to-r from-indigo-500 to-purple-600 text-white rounded-full py-2 px-6 hover:from-indigo-600 hover:to-purple-700 transition-all duration-300"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)